src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/PerformanceAnalyzerPlugin.java

```diff
@@ -233,22 +230,6 @@ public void onIndexModule(IndexModule indexModule) {
         indexModule.addSearchOperationListener(performanceanalyzerSearchListener);
     }
 
-    //follower check, leader check
-    public void onDiscovery(Discovery discovery) {
```

Comment:
I think this was added as part of Fault detector and I believe it was added internationally 

check this PR:
https://github.com/opendistro-for-elasticsearch/performance-analyzer/pull/218

Reply:
@amathur1893 Do we still need this `onDiscovery()` method? 
Since I don't find any reference, also it's currently throw `ClassNotFoundException` when trying to invoke from [LISTENER_INJECTOR_CLASS_PATH](https://github.com/opendistro-for-elasticsearch/performance-analyzer/blob/master/src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/PerformanceAnalyzerPlugin.java#L239)

Reply:
This is only required in AES. The corresponding classes are in AES. This was done to reduce disparity between ODFE PA code and AES PA code. We do require this to publish fault detection metrics - Follower/Leader Check Failure and Latency.